README.md

```diff
@@ -162,11 +162,22 @@ $ balena push DebugFleet
 
 ### 2. Initial Diagnosis
 
-{{>"masterclass/debugging/initial-diagnosis"}}
+The balenaCloud Dashboard includes the ability to run diagnostics on
+a device to determine its current condition. This should
+be the first step in attempting to diagnose an issue without having to
+actually access the device via SSH. Ensuring diagnostics and health checks
+are examined first ensures that you have a good idea of the state a
+device is in before SSHing into it, as well as ensuring that the information can
+be accessed later if required (should a device be in a catastrophic state). This
+helps significantly in a support post-mortem should one be required.
 
 #### 2.1 Device Health Checks
 
-This will trigger a set of [health checks](https://www.balena.io/docs/reference/diagnostics/) to run on the device, and you should see the all the checks as `Succeeded` in the Success column. This shows that the device is healthy and there are no obvious faults. 
+To run diagnostics through balenaCloud dashboard, head to the `Diagnostics` tab in the sidebar and click the `Run checks` button to start the tests.
+
+![diagnostics](https://user-images.githubusercontent.com/22801822/154141814-6953717d-f90a-456b-ad51-474b14dcc5e9.png)
+
+This will trigger a set of [health checks](https://www.balena.io/docs/reference/diagnostics#device-health-checks) to run on the device, and you should see the all the checks as `Succeeded` in the Success column. This shows that the device is healthy and there are no obvious faults. 
```

Comment:
```suggestion
This will trigger a set of [health checks](https://www.balena.io/docs/reference/diagnostics#device-health-checks) to run on the device. You should see all the checks as `Succeeded` in the Success column if the device is healthy and there are no obvious faults. 
```

Reply:
If we don't have it already, we should also add a bit about adding health checks for your own services

Reply:
> we should also add a bit about adding health checks for your own services

you mean we surface custom healthchecks from docker-compose?

Reply:
@rahul-thakoor @myarmolinsky we have it in main docs, maybe missing an example 

> Any checks with names beginning with check_service_ come from user-defined services. These checks interrogate the engine to see if any services are restarting uncleanly/unexpectedly or failing health checks. We allow users to provide their own health checks using the [HEALTHCHECK directive](https://docs.docker.com/engine/reference/builder/#healthcheck) defined in the Dockerfile or docker-compose file. Any health check output will be collected as-is, truncated to 100 characters, and shown as output along with the exit code.

Could be better so I can add in the diagnostics PR